2do_fe/src/redux/reducers/index.js

```diff
@@ -24,13 +24,15 @@ const taskReducer = (state = initialState, action) => {
       };
 
     case types.UPDATE_TASKS_SUCCESS:
+    case types.GET_SINGLE_TASKS_SUCCESS:
       return {
         ...state,
         loading: false,
         singleTask: payload,
       };
 
     case types.GET_ALL_TASKS_FAILURE:
+    case types.GET_SINGLE_TASKS_FAILURE:
     case types.ADD_TASKS_FAILURE:
       return { ...state, loading: false }; // error: payload
 
```

Comment:
Add reducer for Delete Task Request, Success and Failure as well. Other than that Delete function looks good to me!

Reply:
Changes made to fix bugs